.devcontainer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04
 
 ARG TERRAFORM_VERSION=0.14.8
```

Comment:
Could you also change this to 1.0.0? :)

I never use dev containers 🤷🏻‍♂️

Reply:
Will do - and you should use containers, they are great ;-)